File: esports-backend/src/main/java/com/esports/dto/KycStatusResponseDTO.java
package com.esports.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class KycStatusResponseDTO {
    private String verificationLevel;
    private boolean emailVerified;
    private boolean documentUploaded;
    private String documentType;
    private String documentStatus;
}

File: esports-backend-dotnet/Upkart1/DTO/KycStatusResponseDTO.cs
namespace Upkart1.DTO
{
    public class KycStatusResponseDTO
    {
        public string VerificationLevel { get; set; }
        public bool EmailVerified { get; set; }
        public bool DocumentUploaded { get; set; }
        public string DocumentType { get; set; }
        public string DocumentStatus { get; set; }

        public KycStatusResponseDTO(string verificationLevel, bool emailVerified, 
            bool documentUploaded, string documentType, string documentStatus)
        {
            VerificationLevel = verificationLevel;
            EmailVerified = emailVerified;
            DocumentUploaded = documentUploaded;
            DocumentType = documentType;
            DocumentStatus = documentStatus;
        }
    }
}